### PR TITLE
Synopsis fixes

### PR DESCRIPTION
**Pull request by votemike**
Fixes mentioned in https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5738 and https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5800

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/DsAmen/Molecule/Synopsis/_synopsis.scss [hunk 1]
```diff
@@ -17,7 +17,8 @@ $synopsis-toggle-fade-out-time: 0.25s;
     &:checked ~ .synopsis-toggle__long {
         opacity: 0;
         max-height: 0;
-        transition: opacity $synopsis-toggle-fade-out-time linear, max-height 0s linear $synopsis-toggle-fade-out-time;
+        visibility: hidden;
```
Comment (BPScott) [inline src/DsAmen/Molecule/Synopsis/_synopsis.scss]: Does this hide the text for screen readers too?

Comment (votemike) [inline src/DsAmen/Molecule/Synopsis/_synopsis.scss]: ¯\_(ツ)_/¯ How would one go about finding that out?

Comment (votemike) [inline src/DsAmen/Molecule/Synopsis/_synopsis.scss]: Seems screen readers don't read it. http://webaim.org/techniques/css/invisiblecontent/

Comment (BPScott) [inline src/DsAmen/Molecule/Synopsis/_synopsis.scss]: [LMGTFY](https://www.google.co.uk/search?q=visibility+hidden+screen+reader) :p

[The 3rd result](https://www.w3.org/WAI/tutorials/fundamentals/hiding/) sounds like visibility:hidden hides it from screen readers too.

File: tests/DsAmen/Molecule/Synopsis/SynopsisPresenterTest.php [hunk 2]
```diff
@@ -20,6 +20,17 @@ public function testNeedsShorterSynopsis()
         $this->assertFalse($presenter->needsShorterSynopsis());
     }
 
+    public function testNeedsShorterSynopsisMB()
+    {
+        $synopses = new Synopses('অ', 'অং', 'অংশ');
```
Comment (tatitati) [inline tests/DsAmen/Molecule/Synopsis/SynopsisPresenterTest.php]: 😮

**Comment by tatitati** [comment]
Seems ok

File: tests/DsAmen/Molecule/Synopsis/SynopsisPresenterTest.php (hunk 2 above)
Comment (tatitati) [inline tests/DsAmen/Molecule/Synopsis/SynopsisPresenterTest.php]: That means 'Part' (according to google)

**Pull request merged**